packages/mdc-select/foundation.js

```diff
@@ -174,10 +196,12 @@ export default class MDCSelectFoundation extends MDCFoundation {
 
     let maxTextLength = 0;
     for (let i = 0, l = this.adapter_.getNumberOfOptions(); i < l; i++) {
+      const selectBoxAddedPadding = 26;
```

Comment:
Upon further review, it's going to be problematic using the adapter method since it returns the value as a string with the unit attached, e.g.: `"16px"`. Going to assign it to a constant and make a note that it has to stay in sync with a `scss` variable.